### DISCUSSION TITLE
Enable sponsor for other top contributor

### DISCUSSION THREAD

**Discussion by azzamsa** [oae/gnome-shell-pano#302] [Ideas]
Hi 👋🏽 

I see that @Totto16 has contributed a lot lately. If I recall correctly, we can add multiple GitHub usernames to `FUNDING.yml`. Or otherwise, we can use OpenCollective (if this is not overkill).

![image](https://github.com/oae/gnome-shell-pano/assets/17734314/788644f5-9ca7-4d9a-a728-21d3dddc4c51)

Thank you for Pano 🍜

**Comment by oae** [comment]
That is a good idea if @Totto16 has enabled sponsors on GitHub or has an account on another site, we can add it. @Totto16  <3

**Comment by Totto16** [comment]
I activated sponsors here on GitHub. Thank you for this suggestion. ❤️

**Comment by oae** [comment]
Done 
![image](https://github.com/oae/gnome-shell-pano/assets/1043714/dc75ecfe-d7af-4595-a719-023ba0843795)
